forest/shared/src/cli/snapshot_fetch.rs

```diff
@@ -300,6 +405,32 @@ fn filename_from_url(url: &Url) -> anyhow::Result<String> {
     }
 }
 
+fn get_snapshot_name(network: &str, server: &str, filename: &str) -> anyhow::Result<String> {
```

Comment:
This function converts from Filecoin's naming convention to our own naming convention. Maybe change the name of the function to `normalize_filecoin_snapshot_name`. We also need documentation together with examples of valid and invalid input together with the output.

Reply:
fixed